src/models/converters/transactions/mod.rs

```diff
@@ -225,6 +232,7 @@ impl ModuleTransaction {
                 .data_decoded
                 .as_ref()
                 .and_then(|it| it.get_action_count()),
+            is_cancellation: false, // assumption: Module transactions can't be cancellation txs
```

Comment:
That is correct (as module tx don't adjust the nonce). I would remove the comment or add this